test/CommonAdapterTestTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-cache-storage-adapter-test for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-cache-storage-adapter-test/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-cache-storage-adapter-test/blob/master/LICENSE.md New BSD License
+ */
+
+namespace LaminasTestTest\Cache\Storage\Adapter;
+
+use Laminas\Cache\Storage\Adapter\AdapterOptions;
+use LaminasTest\Cache\Storage\Adapter\CommonAdapterTest;
+
+/**
+ * @group      Laminas_Cache
+ */
+final class CommonAdapterTestTest extends CommonAdapterTest
+{
+    public function setUp()
+    {
+        $this->_options = new AdapterOptions();
+        $this->_storage = new TestAsset\MockAdapter($this->_options);
```

Comment:
My purpose here is not to test everything about `CommonAdapterTest` since the resulting mock would be the same as `Memory` adapter, but rather to have a minimum check that it's doing something with the least complex mock.